src/components/GuildCard/index.tsx

```diff
@@ -0,0 +1,130 @@
+import React, { useEffect, useState } from 'react';
+import { ImageBackground } from 'react-native';
+import {
+  Box,
+  Pressable,
+  Heading,
+  Text,
+  VStack,
+  AspectRatio,
+  Spinner,
+  useTheme,
+} from 'native-base';
+import storage from '@react-native-firebase/storage';
+import { Warning } from 'phosphor-react-native';
+
+import { LinearGradientView } from './styles';
+import { GuildDto } from '@interfaces/guild.dto';
+import { CARDS_BORDER_WIDTH, RATIO } from '@utils/constants';
+import firestoreDateFormat from '@utils/fireabseDateFormat';
+import firestoreValueIsValid from '@utils/firestoreValueIsValid';
+
+type Props = {
+  guild: GuildDto;
+};
+
+const GuildCard = ({ guild }: Props) => {
+  const { colors } = useTheme();
+  const [image, setImage] = useState<string | undefined>(undefined);
```

Comment:
can we change to `null` instead of undefined?

Reply:
The react native image component expects a "string | undefined"